### ISSUE TITLE
Get ONLY column names without alias when .stream() 

### ISSUE THREAD

**Issue by anodynos** [typeorm/typeorm#3031]
**Issue type:**

[x] question
[ ] bug report
[x] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[x ] `0.2.7` (or put your version here)

Can we have ONLY the fields without alias prefix when we stream? 

So when [streaming](http://typeorm.io/#/select-query-builder/streaming-result-data) for example: 

```ts
const stream = await getRepository(User)
    .createQueryBuilder("user")
    .where("user.id = :id", { id: 1 })
    .stream();
```

we get a 

```ts
RowDataPacket {
   user_id: 1, 
   user_name: 'Angelos',
   user_dob: '2018-11-01'
}
``` 

all prefixed with `user` which is the alias passed to `createQueryBuilder`. 

But If we omit the alias, we get the name of the class name eg `UserEntityClass_name`, `UserEntityClass_dob` etc. 

This is just a small annoying thing to deal with, but how can we get an object with just the property names, as they are declared in the class?

**Comment by dboune** [comment]
A small addition, as I’ve just run into this.

It would be nice if once could choose to get either the column names as listed in the database, or the column names as the property names from the entity needed to then perform an insert using another entity, as using the actual database column names doesn’t work at the moment, and there appears to not be a “raw” insert mode for the querybuilder.

Currently it seems possible, if one has no entities at all for the tables involved, to work around this in that way.

I think I’ve seen a future plan to have stream return entities, which would be quite useful in some scenarios, but possibly harmful for performance in others.

**Comment by yesworld** [comment]
@pleerock I faced the same problem. What can be done?

**Comment by GeorgeWillaman** [comment]
Same here. Any updates?

**Comment by marcosvvp** [comment]
Just faced this issue and found this question, is there a workaround or any solution?

**Comment by nebkat** [comment]
Ideally it would be possible to stream while mapping to entities (like with `getMany()`), which is something that could be looked in to.

**Comment by marcosvvp** [comment]
> Ideally it would be possible to stream while mapping to entities (like with `getMany()`), which is something that could be looked in to.

Indeed. Quite new to typeorm, but used getMany() a few times and didn't see that alias there. Had to stream() and was the first time I saw the alias_field notation.

**Comment by thangtv611** [comment]
No update from team?

**Comment by sebastiantf** [comment]
Seeing this issue with stream
